### PR TITLE
add WelcomeTextToSpeechPrompt

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/11297

### CODE REVIEW

**Comment by get-itips** [comment]
@CLYVR Can you help me with the parameter description please? Thanks

**Comment by CLYVR** [comment]
@get-itips 
The WelcomeTextToSpeechPrompt parameter indicates the Text-to-Speech (TTS) prompt which is played when callers are connected with the Call Queue.

Type: String
Position: Named
Default value: None
Required: False
Accept pipline input: False
Accept wildard characters: False

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/New-CsCallQueue.md [hunk 1]
```diff
@@ -1183,6 +1183,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -WelcomeTextToSpeechPrompt
+This parameter indicates the Text-to-Speech (TTS) prompt which is played when callers are connected with the Call Queue.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsCallQueue.md:1187]:
```suggestion
This parameter indicates which Text-to-Speech (TTS) prompt is played when callers are connected to the Call Queue.
```

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 2]
```diff
@@ -1200,6 +1200,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -WelcomeTextToSpeechPrompt
+This parameter indicates the Text-to-Speech (TTS) prompt which is played when callers are connected with the Call Queue.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsCallQueue.md:1204]:
```suggestion
This parameter indicates which Text-to-Speech (TTS) prompt is played when callers are connected to the Call Queue.
```

File: skype/skype-ps/skype/New-CsCallQueue.md [hunk 3]
```diff
@@ -1183,6 +1183,21 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
 ### CommonParameters
 This cmdlet supports the common parameters: `-Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).`
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsCallQueue.md:1202]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 4]
```diff
@@ -1200,6 +1200,21 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
 ### CommonParameters
 This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsCallQueue.md:1219]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by msbemba — APPROVED** [review]
Approved

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**